UnityProject/Assets/Scripts/Players/PlayerController.cs

```diff
@@ -1,70 +1,73 @@
 ﻿using System;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
-/* Handles the avatar movement. This script is attached to each of the players.
- */
 
-public class PlayerController : MonoBehaviour 
+namespace Players
 {
-	// General movement variables.
-	private const float speed = 5.0f;
-	private const float moveInterval = 0.5f;
 
-	private float startTime;
+    /* Handles the avatar movement. This script is attached to each of the players.
+     */
 
-	private PlayerData nextState;
-	private Vector2 currPosition;
-	private Vector2 nextPosition;
+    public class PlayerController : MonoBehaviour
+    {
+        // General movement variables.
+        private const float speed = 5.0f;
+        private const float moveInterval = 0.5f;
 
-	// Initialisation.
-	public void Awake()
-	{
-		startTime = Time.time;
-		currPosition = transform.position;
-		nextState = new PlayerData(transform.position);
-	}
+        private float startTime;
```

Comment:
Consider creating a separate package with classes of each category for constants. We seem to use a looooooooooooooot.